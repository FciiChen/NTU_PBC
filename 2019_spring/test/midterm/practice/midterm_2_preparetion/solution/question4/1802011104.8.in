澎湖可以說是台灣暑假的熱門景點，還記得在五月份就有魚粉希望我們能推出澎湖的旅遊文章，沒想到在七月初終於順利成行了！趕緊來和你們分享這次三天兩夜的超充實行程，包準連在夢裡在可以夢到滿滿的鮮蚵～


 


其實魚兒是第三次來到澎湖，搭船與搭飛機方式都有體驗過，不過衡量金錢與時間，怎麼推想都還是搭飛機會比較方便划算一點。有趣的是國內線飛澎湖的航程大概都不需要30分鐘，讓人生第一次搭飛機的柚柚直呼非常不過癮啊，哈哈～

【特米諾1號館】


這幾天民宿選擇的是特米諾1號館，除了距離機場與市區都很近，嶄新溫馨的房間住起來十分舒適，還有提供基本的早餐，像我們一群人包棟也很適合！熱情的店家還會與我們分享澎湖的美食景點，就連口袋名單也不藏私，是初訪澎湖的朋友還蠻不錯的選擇！

【特米諾1號館】
地址：澎湖縣馬公市石泉里185-32號
電話： 06 922 1185
營業時間：24 小時營業
粉絲專頁：特米諾1號館

 

【合發旅行社／金永順旅行社】


來到澎湖到底要怎麼玩最方便？不怕日曬的熱血朋友可以選擇租機車，多款系列機車任你們選擇，機動性也很高。如果有帶長輩或小朋友前來，則會建議你們可以直接租輛汽車；想辦活動或整群朋友來玩，也有適合各人數的巴士可以接送，在澎湖想要趴趴走就是這麼簡單！

【合發旅行社／金永順旅行社】
地址：澎湖縣馬公市臨海路11-1號／澎湖縣馬公市信義路75號
電話： 06 926 6338／06 926 8559
營業時間： 08:00–20:00／08:00–18:00

 

【東海開發育樂】


初次來到澎湖觀光客多會選擇經典的吉貝、七美與望安的跳島行程，若是想要來趟與眾不同的生態美食之旅，不妨可以體驗白灣坑(東海)休閒半日遊！

除了能夠欣賞錠鈎嶼的玄武岩景觀，還能在雞善嶼觀賞燕鷗餵食秀。對了，在與燕鷗近距離拍照時，可要小心隨時可能從天而降的炸彈唷XD



上岸之後就能享用澎湖碳烤鮮蚵無限吃到飽！另外還有供應澎湖海鮮拼盤與海釣活魚現釣現烤，海鮮控來到這裡簡直大解放！吃飽後還能慵懶地躺在平台上享受清涼微風，特別愜意。



每人都還會有一大碗超澎派的澎湖翡翠海鮮粥，盡情地舖上滿滿的鮮蚵幫自己加料就對啦！覺得口渴還有澎湖道地的清涼風茹茶可以無限量暢飲～石斑魚池餵魚秀也是許多親子喜歡體驗的活動，一旁還有專人教學協助。在東海就能享受寓教於樂與海鮮的鮮美，是趟愜意又滿足的半日遊之旅！

【東海開發育樂】
地址：澎湖縣湖西鄉白坑村2-2號
電話：0928 370 086
粉絲專頁：東海開發育樂

【南寮彩繪村/南寮風車有機農場】





結束了海上行程，我們來探訪位於湖西鄉的南寮彩繪村，早期被稱作龜壁社區。這裡除了看得到傳統的聚落樣貌，村內多處可見色彩鮮明的人文彩繪與海龜相關的裝置藝術。整個村子不會很大，大概慢慢散步半個小時就能輕鬆逛完。



在地友人還有介紹這間位在保寧宮旁的南寮風車有機農場休息站給我們，裡頭有販售經典的仙人掌冰與小管麵線等諸多餐點。這裡的小管麵線真的是鮮美得完全沒話說，推荐必點！\
BREAK