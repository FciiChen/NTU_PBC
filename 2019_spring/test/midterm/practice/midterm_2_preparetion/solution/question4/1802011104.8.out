澎_的 5
1_館 4
旅_社 4
澎_縣 4
特_諾 4
米_1 4
諾_號 4
來_澎 3
到_湖 3
搭_機 3
